### PR TITLE
Added new audit events for returns and amends

### PR DESCRIPTION
**Pull request by ajchilts**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtaxreturns/audit/Auditor.scala [hunk 1]
```diff
@@ -36,10 +43,98 @@ class Auditor @Inject() (auditConnector: AuditConnector) {
...
+
+  def getPaymentStatementSuccess(internalId: String,
+                                 pptReference: String,
+                                 response: FinancialDataResponse)
+                                (implicit hc: HeaderCarrier, ex: ExecutionContext): Unit = {
+    val payload = GetPaymentStatement(internalId, pptReference, SUCCESS, Some(response), None)
+    auditConnector.sendExplicitAudit(GetPaymentStatement.eventType, payload)
+  }
+
+  def getPaymentStatementFailure(internalId: String,
```
Comment (liampan) [inline app/uk/gov/hmrc/plasticpackagingtaxreturns/audit/Auditor.scala:98]: There is a lot of these, that look very similar, is it possible to make a more generic interface? 
I can see this file over time getting rather crowded

File: conf/application.conf [hunk 2]
```diff
@@ -105,7 +105,7 @@ metrics {
 # Microservice specific config
 
 auditing {
-  enabled = true
+  enabled = false
```
Comment (liampan) [inline conf/application.conf:108]: is this switched back on in env confs?

**Review by liampan — APPROVED** [review]
Approving so can get in to staging and test with Chris, 2 comments to look at though :)

File: conf/application.conf (hunk 2 above)
Comment (ajchilts) [reply conf/application.conf:108]: Yes. Its on in app config base. 👍

File: app/uk/gov/hmrc/plasticpackagingtaxreturns/audit/Auditor.scala (hunk 1 above)
Comment (ajchilts) [reply app/uk/gov/hmrc/plasticpackagingtaxreturns/audit/Auditor.scala:98]: I may merge for Chris then look at some refactoring here. 👍 .

Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtaxreturns/audit/Auditor.scala:98]: yep works for me :)

**Pull request merged**